Trial Updates to Nutraceuticals (#409)
greenelab/covid19-review: content/20.treatments.md

The increased risk of thrombotic complications in COVID-19 infected patients was relatively late to be reported in comparison to other COVID-19 manifestations [@doi:10.1016/j.thromres.2020.04.013; @doi:10.1182/blood.2020006000].
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005; @doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19. 
This may be particularly important for patients already receiving pharmaceutical antiplatelet therapies who may become infected.
This is because there is concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
As a result, the use of other therapeutics such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities may be beneficial and warrant further investigation.

> A recently developed highly purified derivative of EPA known as icosapent ethyl (VascepaTM) [@doi:10.1056/NEJMoa1812792] is currently recruiting participants for randomized controlled clinical trial to investigate its anti-inflammatory activity in COVID-19 positive patients [@clinicaltrials:NCT04412018]. 

Reviewer: ```suggestion
A new clinical trial [@clinicaltrials:NCT04412018] is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified derivative of EPA known as icosapent ethyl (Vascepa ^TM^) [@doi:10.1056/NEJMoa1812792].  
```
Hopefully this is formatted correctly, I haven't done superscripts in Manubot before! I based it on [this](https://github.com/manubot/rootstock/blob/master/content/02.delete-me.md).

> Other randomized controlled trials are in the preparatory stages with the intention of investigating the administration of EPA and other bioactive compounds to COVID-19 positive patients to determine if anti-inflammatory effects or disease state improvements are observed [@clinicaltrials:NCT04335032; @clinical trials:NCT04323228]. 

Reviewer: ```suggestion
Other randomized controlled trials are in the preparatory stages with the intention of investigating the administration of EPA and other bioactive compounds to COVID-19 positive patients to determine whether anti-inflammatory effects or disease state improvements are observed [@clinicaltrials:NCT04335032; @clinicaltrials:NCT04323228]. 
```

Finally, while there have been studies investigating the therapeutic value of n-3 fatty acids against ARDS in humans, there is still limited evidence of their effectiveness [@doi:10.3390/ijms21093104].

**Summary.**
The overall lack of human studies in this area means there is limited evidence as to whether these nutraceuticals could affect COVID-19 infection. 

> Consequently, clinical trials have been proposed and are in the preparatory stages to investigate the anti-inflammatory potential of n-3 PUFA and their derivatives for the treatment of COVID-19.   

Reviewer: ```suggestion
Consequently, clinical trials that have been proposed and are in the preparatory stages will investigate the anti-inflammatory potential of n-3 PUFA and their derivatives for the treatment of COVID-19.   
```

Similarly, a randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation [@doi:10.1093/ajcn/85.3.837].
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a flourescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].

> Currently, there are over ten clinical trials registered with the intention to use zinc in a prevantative or therapeutic manner. 

Reviewer: ```suggestion
Currently, there are over ten clinical trials registered with the intention to use zinc in a preventative or therapeutic manner. 
```

Many of these trials have proposed the use of zinc in conjunction with hydroxychloroquine and azithromycin [@clinicaltrials:NCT04370782; @clinicaltrials:NCT04377646; @clinicaltrials:NCT04395768; @clinicaltrials:NCT04373733]. 

> However, it is not known if these trials will change in light of the lack of evidence supporting the use of hydroxychloroquine.

Reviewer: ```suggestion
However, it is not known how these trials will respond to the emerging lack of evidence supporting the use of hydroxychloroquine.
```
Just an idea of an alternate way to phrase it, I don't know what you think!

Other trials are investigating zinc in conjunction with other nutrients such as vitamin C or n-3 PUFA [@clinicaltrials:NCT04342728; @clinicaltrials:NCT04323228].

**Summary.**

> While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and treatment. 

Reviewer: ```suggestion
Though there is, overall, encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19; thus, the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and/or treatment. 
```

> However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelihood of an infection

Reviewer: ```suggestion
However, given the limited risk, maintaining a healthy diet to ensure an adequate zinc status may be advisable for individuals seeking to reduce their likelihood of infection.
```

Another trial in Italy [@clinicaltrials:NCT04323514] intends to deliver a 10 g infusion of vitamin C to 500 severe COVID-19 patients with pneumonia to assess in-hospital mortality over a 72 hr period as the primary outcome.
The trial is currently recruiting and is due to run until March 2021.
We will not know how effective vitamin C is as a therapeutic for quite some time due to the length of both trials.

> These are not the only trials investigating the potential value of vitamin C as there are currently over fifteen trials registered with clinicaltrials.gov that are either recruiting or are currently in preperation.

Reviewer: ```suggestion
These are not the only trials investigating the potential value of vitamin C, as there are currently (as of June 2020) over fifteen trials registered with clinicaltrials.gov that are either recruiting or are currently in preparation.
```

Reviewer: Just tagging @agitter here, since this seems relevant to #406

This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive. 
There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@clinicaltrials:NCT04334005].

> Other trials are also underway examining various factors including mortality, symptoms recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein (CRP) and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration [@clinicaltrials:NCT04385940; @clinicaltrials:NCT04334005; @clinicaltrials:NCT04411446; @clinicaltrials:NCT04344041]. 

Reviewer: ```suggestion
Other trials are also underway examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein (CRP) and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration [@clinicaltrials:NCT04385940; @clinicaltrials:NCT04334005; @clinicaltrials:NCT04411446; @clinicaltrials:NCT04344041]. 
```
